From 6c737f7ff0069d5b2f116ffaab35217245e9c806 Mon Sep 17 00:00:00 2001
From: alnsantosh <aln.santosh@gmail.com>
Date: Mon, 8 Jan 2018 02:11:34 +0530
Subject: [PATCH 1/2] Testing git

---
 Database.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Database.java b/Database.java
index a1e11f7..0120716 100644
--- a/Database.java
+++ b/Database.java
@@ -1,4 +1,4 @@
-package com.flight.dao;
+package com.flight.dao;//Packages
 
 import java.util.ArrayList;
 import java.util.List;
-- 
2.15.1.windows.2


From cfb1d451a03295c6f70871492f0af30b61b5b840 Mon Sep 17 00:00:00 2001
From: alnsantosh <aln.santosh@gmail.com>
Date: Tue, 9 Jan 2018 14:28:57 +0530
Subject: [PATCH 2/2] testing git

---
 Database.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Database.java b/Database.java
index 0120716..0a9a6e1 100644
--- a/Database.java
+++ b/Database.java
@@ -1,4 +1,4 @@
-package com.flight.dao;//Packages
+package com.flight.dao;//Packages in Database class
 
 import java.util.ArrayList;
 import java.util.List;
-- 
2.15.1.windows.2

